config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```

Comment:
@carols10cents make sure this matches https://github.com/rust-lang/rfcs/pull/1985